iox_data_generator/benches/point_generation.rs

```diff
@@ -0,0 +1,66 @@
+use criterion::{criterion_group, criterion_main, Criterion, Throughput};
+use data_generator::{
```

Comment:
When the package name is changed, this'll need to be changed too:

```suggestion
use iox_data_generator::{
```